test/integration/hooks/custom_hook_01.sh

```diff
@@ -19,4 +19,11 @@ echo "KUBECONFIG=$KUBECONFIG"
 
 cd "$PARENT_DIR/eu-west-1/k8s-qa/services" || return
 
-echo "This hook can be used for new purposes"
+MUTE_ME=$(kubectl get crd ingressroutes.traefik.io --output=custom-columns=NAME:.metadata.name --no-headers)
```

Comment:
Is this variable being used?

Reply:
no, it was just a cheap trick to hide a successfull kubectl run.